openqa-investigate

```diff
@@ -180,6 +180,32 @@ $comment_text"
     fi
 }
 
+post-investigate() {
+    local id=$1 old_name=$2
+    local rc=0 status
+    [[ ! "$old_name" =~ investigate:retry$ ]] && echo "Job is ':investigate:' already, skipping investigation" && return 0
+    result="$(echo "$job_data" | runjq -r '.job.result')" || return $?
+    investigate_origin="$(echo "$job_data" | runjq -r '.job.settings.OPENQA_INVESTIGATE_ORIGIN')" || return 1
+    origin_job_id=${investigate_origin#"$host_url/t"}
+    comment="Investigate retry job: $host_url/t$id"
+    if [[ "$result" == passed ]]; then
+        comment+=" passed, likely a sporadic failure"
+    else
+        comment+=" failed, likely not a sporadic failure"
+    fi
+
+    # meanwhile the original job might have been deleted already, handle
+    # gracefully
+    out=$("${client_call[@]}" -X POST jobs/"$origin_job_id"/comments text="$comment" 2>&1) || rc=$?
+    if [[ "$rc" -ne 0 ]]; then
+        status=$(echo "$out" | runjq .error_status 2>/dev/null || echo "unknown")
+        if [[ "$status" != "404" ]]; then
```

Comment:
Note that there's generally no need to quote variables on the left hand side when using `[[ … ]]`.